channels/README.md

```diff
@@ -334,6 +334,7 @@ channel_settle |   |     open     | -----+
                |   +--------------+
                |    ^ |    |
                |    | | channel_withdraw/
+               |    | | channel_snapshot_solo/
```

Comment:
Reading description, I am not convinced this tx makes channel move to closing.

Reply:
ah these are referred to arrow in the left

Reply:
This suggests that it can be applied only from channel state open, though I do not see this check in [the code](https://github.com/aeternity/epoch/pull/1325/files#diff-ab5fa577fb91bc576556ed547cbef0a4R109)?

Reply:
@velzevur Thanks for pointing out it is checked https://github.com/aeternity/epoch/pull/1325/files#diff-a171f3f67fe181f4f1e61696b31151b9R180